server/src/main/java/org/elasticsearch/index/query/PercolatorExecutionContext.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.index.query;
```

Comment:
Can this live in the percolator module?

Reply:
Apparently not that easy...I have a lot of dependency issues if I try to move it.

Reply:
Will try to move it.